tasks/tasks.go

```diff
@@ -45,19 +57,26 @@ func Sync(ctx context.Context, config *tsbridge.Config) error {
 		return err
 	}
 
-	sd, err := stackdriver.NewAdapter(ctx, config.Options.SDLookBackInterval)
-	if err != nil {
-		return err
-	}
-	defer sd.Close()
+	sdClientOnce.Do(func() {
+		sdClient, err = stackdriver.NewAdapter(ctx, config.Options.SDLookBackInterval)
+		if err != nil {
+			errSync = append(errSync, fmt.Errorf("Unable to initialize stackdriver adapter: %v", err))
```

Comment:
readability nit: error strings should not be capitalized. same for other error strings in this func.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Fixed!